app/_assets/stylesheets/badges.less

```
@@ -28,6 +28,13 @@
       text-fill-color: transparent;
     }
   }
+  &.events::after {
+    content: "Supports Events";
+    color: white;
+    background: linear-gradient(90deg, #5052FD 0%, #033796 100%);
+    padding: 5px 12px 4px;
+    text-align: center;
+  }
```

```suggestion
  &.events::after {
    content: "Supports Events";
    color: #3972d5;
    background: #f2f6fe;
    padding: 5px 12px 4px;
    text-align: center;
  }
```

Let's simplify this down. This badge style is specific to premium partners and I don't want to muddy that use case.